### DISCUSSION TITLE
conda env for linux U20.04

### DISCUSSION THREAD

**Discussion by sherwoac** [NVlabs/instant-ngp#405] [Ideas / Feature requests]
I created the following [conda](https://docs.conda.io/en/latest/) environment definition for this instant ngp, might be useful to some, welcome to add it to the project if it gets broad adoption. For my on linux/U20.04, instant ngp builds and runs with it.

https://gist.github.com/sherwoac/bad3aef6e3e301b43890bc49fbab6986

**Comment by Tom94** [comment]
Hi there, thank you for this contribution! I do not regularly use Conda but would love to integrate this (or some platform-agnostic form of it) into the codebase. Would you mind creating a pull request so it doesn't fall through the cracks? I'd then have a closer look, shuffling the file around, at an opportune time in the future. Cheers!

**Reply by sherwoac** [reply]
Sure. Will do, next week.

**Comment by luchaoqi** [comment]
Any updates on this? The file in the link above no longer works.

**Comment by sherwoac** [comment]
sure, I can have a go, can you help test it?

**Reply by luchaoqi** [reply]
Sure, would love to help test it.

**Comment by sherwoac** [comment]
you can find it on my fork, added section to the README.md and
environment.yml:

https://github.com/sherwoac/instant-ngp/

Lemme know if you have any difficulties.

Thanks.

Adam

**Reply by luchaoqi** [reply]
Still having some problems with pkg versions but can be solved through `conda update --all`. Then it prompts `CUDA_ARCHITECTURES is empty for target "cmTC_efa1e"` which is similar here: [[NVlabs/instant-ngp: Instant neural graphics primitives: lightning fast NeRF and more](https://github.com/NVlabs/instant-ngp#troubleshooting-compile-errors)](https://github.com/NVlabs/instant-ngp#troubleshooting-compile-errors)

**Comment by sherwoac** [comment]
Hi.

I don't specify update in the README.md, haven't tried it, so can't say
what would happen. did you try without?

Adam

**Reply by luchaoqi** [reply]
Think I finally figure out using NVIDIA's cudatoolkit for conda env and some tips here: [NVlabs/instant-ngp: Instant neural graphics primitives: lightning fast NeRF and more](https://github.com/NVlabs/instant-ngp#troubleshooting-compile-errors). Thanks!
